keyboards/dztech/dz60rgb/dz60rgb.c

```diff
@@ -725,32 +641,29 @@ const rgb_led g_rgb_leds[DRIVER_LED_TOTAL] = {
     {{4|(5<<4)},    { 17.23*5,  64}, 0},
     {{4|(2<<4)},    { 17.23*2,  64}, 1},
     {{4|(1<<4)},    { 17.23*1,  64}, 1},
-    {{4|(0<<4)},    { 17.23*0,  64}, 1}
-
+    {{4|(0<<4)},    { 17.23*0,  64}, 1},
 };
 #endif
 
 void matrix_init_kb(void) {
     matrix_init_user();
 }
-
-
 void matrix_scan_kb(void) {
     matrix_scan_user();
 }
-
-
 bool process_record_kb(uint16_t keycode, keyrecord_t *record) {
     return process_record_user(keycode, record);
 }
-
-
 void suspend_power_down_kb(void)
 {
     rgb_matrix_set_suspend_state(true);
+    suspend_power_down_user();
+    rgb_matrix_disable();
 }
 
 void suspend_wakeup_init_kb(void)
 {
     rgb_matrix_set_suspend_state(false);
+    suspend_wakeup_init_user();
+    rgb_matrix_enable();   
```

Comment:
rgb_matrix_enable shouldn't be necessary as rgb_matrix_set_suspend_state will have the same effect.

Reply:
correction, rgb_matrix_set_suspend_state requires RGB_DISABLE_WHEN_USB_SUSPENDED defined true